### ISSUE TITLE
validation

### ISSUE THREAD

**Issue by GYC1996** [chenjun2hao/CenterFace.pytorch#25]
can you get me the annotation of the validation set. Thank you

**Comment by chenjun2hao** [comment]
the package of all the data can download from:https://pan.baidu.com/s/1rB4rd53byGmtD_H2AShs-Q 提取码: 3i6b
